<?xml version="1.0"?>
<launch>
    <param name="use_sim_time" value="false" />
    <!--<arg name="calibration_file_position" value="$(find undistort_images)/calib_files/stereo.yaml"/>-->
    <arg name="left_info_topic" value="/wide/left/camera_info"/>
    <arg name="right_info_topic" value="/wide/right/camera_info"/>
    <arg name="left_image_topic" value="/wide/left/image_raw"/>
    <arg name="right_image_topic" value="/wide/right/image_raw"/>
    <arg name="queue_size_img" value="1"/>
    <arg name="queue_size_imu" value="10"/>


    <node pkg="undistort_images" type="publishStereoCameraInfo" name="publishStereoCameraInfo" output="screen">
        <param name="topic/left_info" value="$(arg left_info_topic)" />
        <param name="topic/right_info" value="$(arg right_info_topic)" />
        <param name="topic/left_img" value="$(arg left_image_topic)" />
        <param name="topic/right_img" value="$(arg right_image_topic)" />
        <param name="camfile_left" value="stereo_left" />
        <param name="camfile_right" value="stereo_right" />
        <param name="queue_size/img" value="$(arg queue_size_img)" />
        <param name="queue_size/imu" value="$(arg queue_size_imu)" />
        <!--<param name="calibration" value="$(arg calibration_file_position)" />-->
    </node>
</launch>
